### PR TITLE
Improve existing emblems and add new

### PR DESCRIPTION
**Pull request by eaglersdeveloper**
I improved existing emblems to match one style and add new.

I made some screenshots. Take a look:

   

Closes https://github.com/ubuntu/yaru/issues/1039

### CODE REVIEW

**Comment by clobrano** [comment]
Black symbolic link looks a bit too bold. Are there different color
options?

**Comment by Feichtmeier** [comment]
image
Share works

**Comment by eaglersdeveloper** [comment]
> Black symbolic link looks a bit too bold. Are there different color options?

I think that dark is the perfect color for symlink emblem. 👌

**Comment by Feichtmeier** [comment]
Just Tone it down a bit eaglers
Make it $jet

**Comment by madsrh** [comment]
Nice work @eaglersdeveloper 👍 
I do agree with @clobrano that's it's a bit heavy. Would it be possible to invert it to give it a lighter feeling?

image

image

**Comment by madsrh** [comment]
I don't know what the rules regarding shadow for suru icons are, but I can see that there's a subtle shadow below the icon and I was wondering if we could add a bit more, like we did with the batches, to makes it look as if it's floating above? 

image
image

**Comment by Paz-it** [comment]
TBH I'm not sure we need to replace the arrow : 
screenshot from 2019-02-04 06-15-34

**Comment by clobrano** [comment]
Old arrow looks outdated for me.

https://github.com/ubuntu/yaru/pull/1151#issuecomment-460092248

I prefer the gray arrow

**Comment by eaglersdeveloper** [comment]
@Feichtmeier
> Just Tone it down a bit eaglers
> Make it $jet

I don't see a fundamental difference, but I still commit.

**Comment by clobrano** [comment]
> @Feichtmeier
> 
> > Just Tone it down a bit eaglers
> > Make it $jet
> 
> I don't see a fundamental difference.

You can just post a comparison, if you prefer. What about @madsrh  suggestion?

**Comment by eaglersdeveloper** [comment]
ezgif-4-4c0a1e563149

**Comment by madsrh** [comment]
@eaglersdeveloper could you try [the white version](https://github.com/ubuntu/yaru/pull/1151#issuecomment-460092248)?

**Comment by Feichtmeier** [comment]
Concerning shadows: I guess that's rather a nautilus setting. Maybe that's style able, maybe not

**Comment by eaglersdeveloper** [comment]
I don't think, because the icon has a shadow. It disappeared on GIF.

**Comment by Paz-it** [comment]
@eaglersdeveloper , awosome work! - the only one is the arrow too dark.
Can you try  @madsrh idea? - White background with black arrow?

**Comment by eaglersdeveloper** [comment]
🤔

**Comment by Feichtmeier** [comment]
@eaglersdeveloper I think Mads is right and we should try the colour invasion, could you please try that?

**Comment by eaglersdeveloper** [comment]
Then what is the meaning of the circle, if it is the same color as the background?
OK, I will try.

**Comment by Feichtmeier** [comment]
It looks much better - thank you for commiting. Much more neutral and less heavy on the eyes. Could you also commit the PNGs?

**Comment by madsrh** [comment]
Okay @eaglersdeveloper I've tried the inverted colors and it doesn't really work. Let's go back to the white arrow on a black circle. Could you try with one of these: 

**#2d2d2d**
image

**#3d3d3d**
image

**For reference, the initial version:** 
image

**Comment by clobrano** [comment]
+1 for #3D3D3D

**Comment by clobrano** [comment]
@eaglersdeveloper, if you can make it by lunch it would be great. I am going to release on stable this afternoon :+1:

**Comment by eaglersdeveloper** [comment]
> I've tried the inverted colors and it doesn't really work.

I forgot render raster icons. Perhaps because of this didn't work.

**Comment by madsrh** [comment]
@eaglersdeveloper when I say it doesn't work I just mean that it doesn't look very good. Bad suggestion, sorry

**Comment by eaglersdeveloper** [comment]
Nothing wrong!

I for current.

**Comment by clobrano** [comment]
> Nothing wrong!
> 
> I for current.

You do understand that we are reviewing this PR, right? :smile: 

The white circle, as you anticipated actually, blends too much with a light background
photo5965387980334018261

the picture above is not even the worst use case, imaging a directory full of files and no directories (so all white), a few symlinks won't be visible enough

**Comment by eaglersdeveloper** [comment]
So who is for black, who is for white?

👍 for black
👎 for white
😕 if you doubt your choice

**Comment by Feichtmeier** [comment]
Black is my heart black is my heart
Please revert the latest commit as your highly representative "poll" shows the winner ;)

**Comment by madsrh** [comment]
My +1 is for #3D3D3D

**Comment by eaglersdeveloper** [comment]
And `#2d2d2d`, and `#3d3d3d` looks bad in dark theme.

Current, `#2d2d2d`, `#3d3d3d`:

**Comment by clobrano** [comment]
I tested #3D3D3D in dark variant, it looked good. Why do you think the opposite?

**Comment by eaglersdeveloper** [comment]
Window background color in dark theme is also `#3d3d3d`

**Comment by Feichtmeier** [comment]
@eaglersdeveloper that was not "git revert ... "

https://github.com/ubuntu/yaru/pull/1151/commits/9efbf9845a6d5a50a4fa51ca7202bda264925989?short_path=5f5d7f0#diff-5f5d7f0cdf2eb383f1d37b4326c01e61

https://github.com/ubuntu/yaru/pull/1151/commits/a13dceb9d57c2115fb7c06bf41115bd5ca8ba414?short_path=c0f2e41#diff-c0f2e41ddd71c40055fc02daaf6e63f3

**Comment by eaglersdeveloper** [comment]
Now you can merge it.

**Comment by Feichtmeier** [comment]
It's always a pleasure to chat with you! 
I test it again this Eve (or maybe mads or Carlo are faster)

File: icons/src/fullcolor/emblems/emblem-symbolic-link.svg [hunk 1]
```diff
@@ -523,7 +523,7 @@
          height="15"
          width="15"
          id="rect4414"
-         style="display:inline;fill:#f9f9f9;fill-opacity:1;stroke:none;stroke-width:1;stroke-opacity:0.25098039;enable-background:new"
+         style="display:inline;fill:#1a1a1a;fill-opacity:1;stroke:none;stroke-width:1;stroke-opacity:0.25098039;enable-background:new"
```
Comment (clobrano) [inline icons/src/fullcolor/emblems/emblem-symbolic-link.svg]: This is supposed to be #3D3D3D, not #1A1A1A

Comment (eaglersdeveloper) [reply icons/src/fullcolor/emblems/emblem-symbolic-link.svg]: I indicated the reason for choosing a darker color in https://github.com/ubuntu/yaru/pull/1151#issuecomment-460772929

**Comment by eaglersdeveloper** [comment]
> Why did you also change the share emblems?

I align the pictogram.

File: icons/src/fullcolor/emblems/emblem-symbolic-link.svg (hunk 1 above)
Comment (clobrano) [reply icons/src/fullcolor/emblems/emblem-symbolic-link.svg]: So you made a poll, your idea lost, and you ignored it?

Comment (clobrano) [reply icons/src/fullcolor/emblems/emblem-symbolic-link.svg]: Moreover you used the color that was rejected by anyone, nice move :+1:

Comment (eaglersdeveloper) [reply icons/src/fullcolor/emblems/emblem-symbolic-link.svg]: Sorry, I didn't understand you.

Comment (eaglersdeveloper) [reply icons/src/fullcolor/emblems/emblem-symbolic-link.svg]: @Feichtmeier agreed with me that window background color in dark theme is also `#3d3d3d`. If I change emblem color to this color, then the circle become less noticeable in dark theme.

**Comment by madsrh** [comment]
Okay, we have to find a color that works on both light and dark background. Could we try Graphite **#666666** ?

image

image

**Comment by eaglersdeveloper** [comment]
We so will endlessly choose the color :confused:

**Comment by eaglersdeveloper** [comment]
Let's merge this PR now. We will discuss about emblem color later.

**Comment by clobrano** [comment]
> Let's merge this PR now. We will discuss about emblem color later.

There is no point in merging this now if the PR is not ready

**Comment by madsrh** [comment]
@eaglersdeveloper are you okay with #666666? and if so, can update the PR?

**Comment by Paz-it** [comment]
Let me add some(lemon) juice  to the menu ..I'm still +1 for #3D3D3D

**Comment by Feichtmeier** [comment]
@eaglersdeveloper could you please push a complete commit (svg+pngs) with graphite?

**Comment by Feichtmeier** [comment]
@eaglersdeveloper we merge this now, but your behavior is really hard to understand sometimes. If you want to force your opinion you just stop to reply. Maybe you think that is a great idea, but it is not. 
Look, I am also not "human's best friend" - but collaboration includes communication with other humans.

**Comment by eaglersdeveloper** [comment]
😮 I already wanted to accept `#2d2d2d` color

**Comment by clobrano** [comment]
> There is no point in merging this now if the PR is not ready
> @eaglersdeveloper are you okay with #666666? and if so, can update the PR?
> @eaglersdeveloper could you please push a complete commit (svg+pngs) with graphite symbolic link emblems?

The request was different from #2d2d2d. Now, we understand one cannot always be online and responsive, but this is not the first time you act like this, so it gave us some doubts

**Comment by eaglersdeveloper** [comment]
I think that better to use symlinks to `audio-x-generic` for all audio formats.

**Pull request merged**